src/zenml/integrations/pytorch_lightning/materializers/pytorch_lightning_materializer.py

```diff
@@ -40,15 +42,16 @@ def load(self, data_type: Type[Any]) -> Trainer:
             A PyTorch Lightning trainer object.
         """
         super().load(data_type)
-        return Trainer(
```

Comment:
Was this even working before? 🤔

Reply:
I honestly doubt it, I don't see how the trainer will be able to load the trainer without the class definition